proto/ibc/applications/fee/v1/metadata.proto

```diff
@@ -0,0 +1,16 @@
+syntax = "proto3";
+
+package ibc.applications.fee.v1;
+
+option go_package = "github.com/cosmos/ibc-go/v3/modules/apps/29-fee/types";
+
+import "gogoproto/gogo.proto";
+
+// Metadata defines the ICS29 channel specific metadata encoded into the channel version bytestring
+// See ICS004: https://github.com/cosmos/ibc/tree/master/spec/core/ics-004-channel-and-packet-semantics#Versioning
+message Metadata {
+  // version defines the ICS29 fee version
+  string version = 1;
```

Comment:
I would prefer we make this explicit and call this `fee_version`

Reply:
Done!